debian/cuttlefish-common.cuttlefish-host-orchestrator.init

```diff
@@ -53,7 +53,15 @@ function gen_cert {
     mkdir -p "${CERT_DIR}"
     pushd "${CERT_DIR}"
     # https://go.dev/src/crypto/tls/generate_cert.go
```

Comment:
remove this line if using openssl for cert generation.

Reply:
Done